release-notes/2.1/2.1.1-commit.md

```diff
@@ -35,4 +36,45 @@ This is a somewhat currated list of commits across the entire .NET Core 2.1.1 de
 
 * [`[0a99dd88]`](https://github.com/dotnet/wcf/commit/0a99dd88) Add net461 as a supported framework for S.SM.Security.
 * [`[45855085]`](https://github.com/dotnet/wcf/commit/45855085) Generate ThisAssembly.cs, update the version and links for svcutil.xmlserializer (#2893)
-* [`[68457365]`](https://github.com/dotnet/wcf/commit/68457365) Target svcutil.xmlserializer app at dotnetcore. (#2855)
\ No newline at end of file
+* [`[68457365]`](https://github.com/dotnet/wcf/commit/68457365) Target svcutil.xmlserializer app at dotnetcore. (#2855)
+
+## ASP
+
+### Entity Framework
+
+* [`[5bf9cc37]`](https://github.com/aspnet/EntityFrameworkCore/commit/5bf9cc370699f86e3609c183b3b23bfe2e3b1b23) Adds recursion guards to analyzer. [(#12054)](https://github.com/aspnet/EntityFrameworkCore/issues/12054)
+* [`[ef61273e]`](https://github.com/aspnet/EntityFrameworkCore/pull/12164/commits/ef61273ed948a2be8f0a081dc57b16ad3bc2de03) [(#12119)](https://github.com/aspnet/EntityFrameworkCore/issues/12119)
+* [`[1d447bdb]`](https://github.com/aspnet/EntityFrameworkCore/commit/1d447bdb0a550c52a04dcbd2cbc1200faa837686) SqlServer RevEng: Use concat operators instead of function[(#11932)](https://github.com/aspnet/EntityFrameworkCore/issues/11932)
+
```

Comment:
@leecow Looks like the two EF issues I commented on have now been removed. If this is all the commits in 2.1.1, then these two issues should still be here; they just had the wrong commits associated before.

Reply:
I think the changes are largely captured by [7ce9270ac7] Fix Sockets hang caused by concurrent Socket disposal (#29786) (#29846)